src/include/homestore/index/index_internal.hpp

```diff
@@ -64,29 +64,70 @@ enum class index_buf_state_t : uint8_t {
 };
 
 ///////////////////////// Btree Node and Buffer Portion //////////////////////////
+
```

Comment:
General comment, it would be very useful if we put a README under wb_cache directory to illustrate the hierarchy of btree_node -> IndexBuffer and NodeBuffer relationship and its clean/dirty copy behavior.

Reply:
And it will be super helpful if we can explain first, second and third meaning with an example, so that after several month we can read the code easier. :)